blueocean-dashboard/src/main/js/components/karaoke/rest/KaraokeApi.js

```diff
@@ -62,9 +84,7 @@ export class KaraokeApi {
         const fetchOptions = prepareOptions();
         const href = generateDetailUrl(pipeline, branch, runId);
         logger.debug('Fetching href', href);
-        return Fetch.fetchJSON(href, { fetchOptions })
-            .then(FetchFunctions.checkStatus)
-            .then(data => capabilityAugmenter.augmentCapabilities(data));
```

Comment:
I understand the first drop `.then(FetchFunctions.checkStatus)` but not `.then(data => capabilityAugmenter.augmentCapabilities(data))` AFAIR that is needed but did not looked at the code for a while

Reply:
@scherler `Fetch.fetchJSON` does it internally, it's not necessary.

Reply:
`.then(data => capabilityAugmenter.augmentCapabilities(data))` no that is not done in `Fetch.fetchJSON`

Reply:
What is this then, if it’s not the same thing? https://github.com/jenkinsci/blueocean-plugin/blob/master/blueocean-core-js/src/js/fetch.js#L290

Reply:
You are right I confused it with https://github.com/jenkinsci/blueocean-plugin/blob/master/blueocean-dashboard/src/main/js/components/karaoke/services/Augmenter.js sorry